README.md

```diff
@@ -100,8 +100,42 @@ export default class MyForm extends Component {
 }
 ```
 
+#### Example with `watch` argument
+
+To validate the form state on initial render add the `'watch'` argument with the value `true`.
+
+```hbs
+  <input {{validity
+    (fn this.matchTo this.match)
+    on="change"
+    watch=true
+  }}>
+```
```

Comment:
Are you suggesting that a different API for initial validation would be better then `watch=true`?

What should I do to unblock this?

Reply:
Maybe `on` could accept `insert` as a special event type (akin to how `validator-update` was a special event type)?

Reply:
I feel like I should be more explicit about this. I'm proposing the use of `validateDidInsert=true` instead of `watch=true` and perhaps leave `watch` for use with the autotracking/reactive-validating aspect. Also I'm considering `validateTracked=(array this.match1 this.match2)` instead of `watch`.